cbcdb/main.py

```diff
@@ -0,0 +1,375 @@
+import time
+from typing import List, Any, Dict, Tuple
+
+import pandas as pd
+from numpy import inf
+from psycopg2 import connect
+from psycopg2.extras import execute_values
+from sshtunnel import open_tunnel
+
+from cbcdb.configuration_service import ConfigurationService as Config
+
+
+class DBManager:
+    """
+    DBManager handled the read and write information to the DB.
+    This class handles the transport of data to and from the db. The goal is to use
+    highly reusable methods, shareable across most of the classes.
+    """
+
+    def __init__(self,
```

Comment:
This is a minor point, but I would prefer to keep the overriding of config variables in the `__init__` method of the ConfigurationService. Then just access the variables via through the service (i.e. using `self._config.var_name`). I think that would help separate concerns between the ConfigurationService and the DBManager.